src/Sources/Code/Module.cpp

```diff
@@ -206,11 +224,12 @@ void SendNewClientResponse(Client* client, Client* nc) {
 // and send them to the SimConnect client
 void ListLVars(Client* client) {
 	int lVarId = 0;
+	(void)lVarId;
```

Comment:
what does this exactly do?

Reply:
This is an unused Variable, which causes a compiler warning. I didn't want to change the original code, but get rid of the compiler warning for a better overview. If we agree, that the variable is and will not be used, I guess the proper way would be to remove it completely.

Reply:
we should remove it then. No problem.